content/en/apps/tutorials/death-reporting.md

```diff
@@ -0,0 +1,101 @@
+﻿# Building A Death Report Workflow
+
+#### Guide for setting up a comprehensive death report workflow.
+
+This tutorial will take you through setting up a comprehensive death report workflow. This includes laying out a death report form as well as handling all the configurations needed for wiring it up in the CHT.
+By the end of the tutorial you should be able to:
+
+- View the death report menu
```

Comment:
worth mentioning how to make somebody "undead"?

Reply:
Mhm, I think it should be an option actually. For Muso it might be very specific not to make people undead but for another config, this may be needed. Let me added anyway and mention that it can be optional.